.circleci/images/test-image-arm/Dockerfile

```diff
@@ -0,0 +1,63 @@
+# Copyright 2018-2019 the u-root Authors. All rights reserved
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+FROM circleci/golang:1.11
+
+# Install dependencies
+RUN sudo apt-get update &&                          \
+    sudo apt-get install -y --no-install-recommends \
+        `# Linux dependencies`                      \
+        git                                         \
+        bc                                          \
+        bison                                       \
+        flex                                        \
+        gcc-arm-linux-gnueabi                       \
+        make                                        \
+        `# QEMU dependencies`                       \
+        libglib2.0-dev                              \
+        libfdt-dev                                  \
+        libpixman-1-dev                             \
+        zlib1g-dev                                  \
+        `# Linux kernel build deps`                 \
+        libelf-dev &&                               \
+    sudo rm -rf /var/lib/apt/lists/*
```

Comment:
why do you remove this?

Reply:
Without it, the docker container is a few 100MB larger which takes longer to upload.